src/pdfnorm.jl

```diff
@@ -0,0 +1,58 @@
+export pdfL2norm
+
+"""
+    pdfL2norm(d::Distribution)
+
+Return the L2 norm of the probability distribution function `f(x)` of the distribution `d`:
+
+```math
+\\int_{S} f(x)^{2} \\mathrm{d} x
+```
+
+where `S` is the support of `f(x)`.
+"""
+pdfL2norm
+
+pdfL2norm(d::Distribution) = throw(ArgumentError("L2 norm not implemented for $(d)"))
+
+function pdfL2norm(d::Beta{T}) where T
+    if d.α > 0.5 && d.β > 0.5
+        return beta(2 * d.α - 1, 2 * d.β - 1) / beta(d.α, d.β) ^ 2
+    else
+        return T(Inf)
+    end
+end
+
+pdfL2norm(d::Cauchy) =  1 / (d.σ * 2 * π)
```

Comment:
Here one can also use StatsFuns:
```suggestion
pdfL2norm(d::Cauchy) = inv(twoπ * d.σ)
```